dace/config_schema.yml

```diff
@@ -29,14 +29,22 @@ required:
                     Automatically specialize every SDFG to the symbol values 
                     at call-time. Requires all symbols to be set.
 
-            interface:
+            autooptimize:
```

Comment:
Isn't this misleading? Auto-optimize sounds like DaCe will just make your program fast. Maybe just "optimizer" or "cli_optimizer" or...

Reply:
You can still choose the optimizer in the next configuration to not be CLI (we use this in the transformation tester), so "cli_optimizer" is out. can we call it "optimize_on_call"?

Reply:
Maybe something with "transform" is also more appropriate than "optimize"?

Reply:
Done

Reply:
I still don't like that it's called "autooptimize" for the above reason :spooderman:

Reply:
But it's now called transform_on_call